### PR TITLE
New body heading

### PR DESCRIPTION
**Pull request by iamkeir**
Adding a heading that matches the body.

### CODE REVIEW

File: src/components/raspberry-pi/type/body-heading/_index.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@use 'src/mixins/margin';
+@use 'src/primitives/heading';
+@use 'src/mixins/type-scale';
+@use 'src/tokens/line-height';
+
+.sc-rp-type-body-heading {
+  @include heading.primitive;
+  font-size: inherit;
```
Comment (Jonic) [inline src/components/raspberry-pi/type/body-heading/_index.scss:8]: I'd rather we were more explicit about setting the values of things. I've had issues in the past where `inherit` can tie in knots, and `initial` is far too vague. If we can set this to `skateboard` that would be better I reckon!

**Review by Jonic — COMMENTED** [review]
Not sure how I'm feeling about this name. I get that it's a "heading that has the same font size as the body", but when you think about it EVERYTHING is in the body, including all the other headings.

We could do with looking again at the headings in general (following on from our call earlier), so for now I'd be reluctant to introduce and use a new heading before we've had chance to review the lot. I reckon we'll end up moving them all out to generic components, and it would be my initial thinking that this "body" style would end up being the primitive, which in turn would be just ... what you get when you chuck a `h1-6` on the page, without it being something we specifically ask for.

I reckon for now let's not do this, but keep it in mind for a bit of work to address the type system as a whole. In the meantime you can add this to the JAMstack app as like `c-type-base-heading` or something, rather than as a Sauce component. Sound good?

File: src/components/raspberry-pi/type/body-heading/_index.scss (hunk 1 above)
Comment (iamkeir) [reply src/components/raspberry-pi/type/body-heading/_index.scss:8]: @Jonic happy to do that although Greg's brief was this heading should always match whatever the body is. With the above, we create a dependency between this and the body in terms of keeping them 'in sync', if the body should ever change. If you're happy with that, I'll do it.

Comment (Jonic) [reply src/components/raspberry-pi/type/body-heading/_index.scss:8]: Yeah we'll handle how we keep `font-size`s in sync when we're introducing the new type and spacing system. We can't really make the changes to the type scaling system without also addressing the type components themselves, so we'll definitely make sure it's all covered there. As it currently stands, the `heading` primitive says nothing about font-size, so the `font-size` will always match that of the `<body>` anyway.

Does this throw any spanners in the works for your JAMstack work?

Comment (iamkeir) [reply src/components/raspberry-pi/type/body-heading/_index.scss:8]: @Jonic all gravy - let's ditch this PR for now.

**Pull request closed without merging**